hyperapp.d.ts

```diff
@@ -0,0 +1,149 @@
+declare module "hyperapp" {
+  export interface VNodeData {
+    [key: string]: any
+  }
+
+  export interface VNode {
+    tag: string
+    data: VNodeData
+    children: string | VNode | Array<VNode>
+  }
+
+  export type viewFunction<State, Actions> = (
+    state: State,
+    actions: Actions
+  ) => VNode
+  export type emitFunction<Data> = (event: string, data?: Data) => Data
+  export type vnodeFunction = (
+    props: VNodeData,
+    children: string | VNode | Array<VNode>
+  ) => VNode
+
+  export interface Actions<State> {
+    [key: string]:
+      | ((
+          state?: State,
```

Comment:
I feel like this is begging for a named interface. TypeScript produces better error messages against named interfaces rather than anonymous types.